dev-tools/vagrant_scripts/winProvision.ps1

```diff
@@ -58,7 +58,19 @@ if (-Not (Get-Command "python" -ErrorAction SilentlyContinue)) {
 }
 
 echo "Updating pip"
-python -m pip install --upgrade pip 2>&1 | %{ "$_" }
+For ($i = 0; $i -lt 5) {
```

Comment:
I'm not sure whether this script is ever used. If I recall the python specific testing thing runs within the Mage build system that runs in the CI

Reply:
🤦 your right.  new change, adding "officially recommended" method for upgrading pip from https://pip.pypa.io/en/stable/installation/#upgrading-pip  This is supposed to get around Windows permission problems with binaries that update themselves.